### DISCUSSION TITLE
How do I fix this?

### DISCUSSION THREAD

**Discussion by Amaranth-101** [community/community#138048] [Programming Help]

**Comment by eldarsji** [comment]
The error shown in the image happens due to the `datetime` module, specifically that datetime is not defined. To fix this, you need to ensure that you have imported the datetime module at the top of the script.

1. If you haven't, add the following line at the top of the script:
`from datetime import datetime`
This will import the datetime class from the datetime module and make it available for use in the code.

2. Alternatively, if you want to use the entire datetime module, you can write:
`import datetime`
However, in that case, you will need to modify your code to access datetime as datetime.datetime.now() instead of datetime.now().

Once you add the appropriate import statement, the error should be resolved.
